proto/engine/v1/electra.go

```diff
@@ -1,4 +1,118 @@
 package enginev1
 
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
 type ExecutionPayloadElectra = ExecutionPayloadDeneb
 type ExecutionPayloadHeaderElectra = ExecutionPayloadHeaderDeneb
+
+var (
+	drExample = &DepositRequest{}
+	drSize    = drExample.SizeSSZ()
+	wrExample = &WithdrawalRequest{}
+	wrSize    = wrExample.SizeSSZ()
+	crExample = &ConsolidationRequest{}
+	crSize    = crExample.SizeSSZ()
+)
```

Comment:
This works and it's cheap but it's ugly: we are allocating a useless request just to get the ssz size, don't we have already these static sizes computed?

Reply:
it's dynamic right? just making sure it's the same always if we regenerate

Reply:
I meant these can easily be compile time constants

Reply:
if you care about the ~1k wasted by these values, a compromise where you don't need to hardcode the sizes would be to use an init func:
```
var drSize, wrSize, crSize int

func init() {
	exDr, exWr, exCr = &DepositRequest{}, &WithdrawalRequest{}, &ConsolidationRequest{}
	drSize, wrSize, crSize = exDr.SizeSSZ(), exWr.SizeSSZ(), exCr.SizeSSZ()
}
```